db/migrate/20170725153927_create_data_requests.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDataRequests < ActiveRecord::Migration[5.1]
+  def change
+    create_table :data_requests, id: :uuid do |t|
+      t.integer :user_id
+      t.integer :workflow_id
```

Comment:
is this in place of declaring a workflow_id column explicitly? not as used to rails as i'd like to be, and not sure how to generate a migration to change this relation without damaging existing records